tests/unit/lms/services/assignment_test.py

```diff
@@ -357,14 +354,16 @@ def non_matching_params(self, request, matching_params):
 
     @pytest.fixture(autouse=True)
     def with_assignment_noise(self, assignment):
-        factories.Assignment(
-            tool_consumer_instance_guid=assignment.tool_consumer_instance_guid,
-            resource_link_id="noise_resource_link_id",
-        )
-        factories.Assignment(
-            tool_consumer_instance_guid="noise_tool_consumer_instance_guid",
-            resource_link_id=assignment.resource_link_id,
-        )
+        return [
```

Comment:
This fixture now not only creates the assignments but returns them. I reckon all fixtures should return the values they create even if not immediately used.